extensions/sql-migration/src/constants/queries.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+//Select all databases available on the server with size
+export const SELECT_ALL_DB_W_SIZE = `
```

Comment:
Let's double check this query with PMs real quick, but this looks much better than the existing one!